pyproject.toml

```diff
@@ -66,6 +67,9 @@ pandas-stubs = "^2.0.0.230412"
 types-pillow = "^9.5.0.2"
 
 
+[tool.poetry.extras]
+production = ["langchain-serve"]
```

Comment:
I've added a new extra `production` with langchain-serve. Please advise if a different name is advised here. This would need changes to [the workflow in langchain-serve](https://github.com/jina-ai/langchain-serve/blob/76ecf1bc39ab4ef4a4b3f94f30411b4b44e0b03d/.github/workflows/langflow-push.yml#L55)

Reply:
I think it could be deploy

Reply:
Changed to `deploy`